upload-to-test-dashboard/src/config.ts

```diff
@@ -0,0 +1,14 @@
+import * as env from "env-var";
+
+export const config = {
+  apiUrl: env.get("INPUT_API_URL").required().asString(),
+  repository: env.get("INPUT_REPOSITORY").required().asString(),
+  branch: env.get("INPUT_BRANCH").required().asString(),
+  testSuite: env.get("INPUT_TEST_SUITE").required().asString(),
+  testFileType: env.get("INPUT_TEST_FILE_TYPE").required().asString(),
+  commitHash: env.get("INPUT_COMMIT_HASH").required().asString(),
+  jobId: env.get("INPUT_JOB_ID").required().asString(),
+  files: env.get("INPUT_FILES").required().asString(),
+  runAttempt: env.get("GITHUB_RUN_ATTEMPT").asString(),
```

Comment:
**question**

Is it an attempt number? Can you share a sample value of it?

Reply:
https://github.com/wunderflats/api/actions/runs/7991859080/job/21852694601?pr=4742
here I tested it in my test PR. You can see there the output of all the env vars, and GITHUB_RUN_ATTEMPT equals 2, that was the first re-run of the same commit, because I simply re-ran the workflow, but chenging the action code.

If you look at the attempt 3 https://github.com/wunderflats/api/actions/runs/7991859080/job/21854432176?pr=4742
it outputs the configuration object we created in the action. runAttampt there is 3.